en/02_Developer_Guides/15_Customising_the_Admin_Interface/08_ReactJS_Redux_and_GraphQL.md

```diff
@@ -952,22 +927,33 @@ Injector.ready(() => {
 
   // Assuming you've got some element in the DOM with the id "notes-app"
   $('#notes-app').entwine({
+    ReactRoot: null,
+
     onmatch() {
-      ReactDOM.render(
+      const root = createRoot(this[0]);
+      this.setReactRoot(root);
+      root.render(
         <ApolloProvider client={apolloClient} store={store}>
           <App />
-        </ApolloProvider>,
-        this[0]
+        </ApolloProvider>
       )
     },
 
     onunmatch: function() {
-      ReactDOM.unmountComponentAtNode(this[0]);
+      const root = this.getReactRoot();
+      if (root) {
+        root.unmount();
+        this.setReactRoot(null);
+      }
     },
   });
 });
 ```
 
+[info]
+`this[0]` is how we get the underlying pure javascript node that the jQuery object is wrapping. We can't pass `this` directly into the `createRoot()` function because react doesn't know how to deal with a jQuery object wrapper.
```

Comment:
Could you, please, peripheries this statement, since `this[0]` returns DOM Element, `pure javascript node` sounds a little bit unusual. Or add a link to JQuery explanation https://api.jquery.com/Types/#jQuery.

Reply:
Changed the wording and added a link to the jquery docs.